### PR TITLE
Added two fields to Etiology. Testing commit and pull.

### PR DESCRIPTION
**Pull request by kathypham-gov**

### CODE REVIEW

File: app/models/patient.rb [hunk 1]
```diff
@@ -5,11 +5,13 @@ class Patient < ActiveRecord::Base
   validates_format_of :name, :with => /\A[^0-9`!@#\$%\^&*+_=]+\z/, message: "No numbers of special chars"
   validates_format_of :ssn, :with => /\d{3}-\d{2}-\d{3}/, message: "Expect format 111-22-3333"
 
-  enum asia_level: (2..8).map { |level| "C#{level}" } + (1..12).map {|level| "T#{level}"} + (1..5).map {|level| "L#{level}"} + (1..3).map {|level| "S#{level}"} + [ :"S4-5" ]
   enum employment_status: [ :Employed, :Unemployed ]
   enum highest_level_of_education: [ :"No Diploma", :"High School", :"Some College", :"Graduate School" ]
   enum registration_status: [ :"Currently Served", :"Not Served" ]
   enum sci_network: { Yes: true, No: false }
+  
+  enum asia_level: (2..8).map { |level| "C#{level}" } + (1..12).map {|level| "T#{level}"} + (1..5).map {|level| "L#{level}"} + (1..3).map {|level| "S#{level}"} + [ :"S4-5" ]
```
Comment (awong-dev) [inline app/models/patient.rb]: why did you move asia_level down a few lines? I was keeping the enums in alphabetical order.

Comment (kathytpham) [inline app/models/patient.rb]: Ah, didn't realize that! Will revert. I did that to organize in alphabetical order by the categories below. Note to self: Never ignore the little voice that says write comments right now and not later.

**Comment by awong-dev** [comment]
Merging for now. Will open issue to fix DB columns and possibly normalize how the fields are listed.
